corehq/apps/export/templates/export/customize_export_new.html

```diff
@@ -111,17 +111,42 @@
 
         </div>
         <div class="form-group">
-          <div class="col-sm-offset-4 col-md-offset-3 col-lg-offset-2 col-sm-4">
+          <div class="col-sm-offset-4 col-md-offset-3 col-lg-offset-2 col-sm-6">
             {% if not export_instance.is_odata_config %}
               <div class="checkbox">
                 <label>
                   <input type="checkbox"
                          id="transform-dates-checkbox"
                          data-bind="checked: transform_dates" />
-                  {% trans "Automatically convert dates and links for Excel" %}
+                  {% if request|toggle_enabled:"EXCEL_EXPORT_DATA_TYPING" %}
+                    <strong>{% trans "Automatically convert dates and multimedia links for Excel" %}</strong><br/>
+                    {% blocktrans %}
+                      Leaving this checked will ensure dates appear in excel format.
+                      Otherwise they will appear as a normal text format. This also allows for
+                      hyperlinks to the multimedia captured by your form submission.
+                    {% endblocktrans %}
+                  {% else %}
+                    {% trans "Automatically convert dates and links for Excel" %}
+                  {% endif %}
                 </label>
               </div>
 
+              {% if request|toggle_enabled:"EXCEL_EXPORT_DATA_TYPING" %}
+              <div class="checkbox">
+                <label>
+                  <input type="checkbox"
+                         id="format-data-in-excel-checkbox"
+                         data-bind="checked: format_data_in_excel" />
```

Comment:
This is minor UX feedback probably for after this is merged and when we're releasing the feature more widely, but this only affects Excel 2007+, right? Maybe it'd make sense to grey/out & disable the checkbox unless Excel 2007+ format is selected?

Reply:
yeah I definitely thought about that. it was a little more of an issue due to how the knockout code for exports is structured. it wasn't super straightforward to add in that logic and would require some digging. If we decide it's worth the extra time, I'm happy to do it.